packages/go_router/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 6.3.0
+
+- Supports for returning values on pop.
+
 ## NEXT
```

Comment:
I meant the ##Next section should be merged in 6.3.0

Reply:
``## Next`` meant whoever publish the next version will need to publish the changes in`` ##NEXT ``as well

Reply:
AHHHHH, got it

Reply:
This seems not address yet?